### DISCUSSION TITLE
AUC0-24h

### DISCUSSION THREAD

**Discussion by WTY-tt** [Open-Systems-Pharmacology/Forum#1479] [Questions & Problems]
Hello, I'm modeling with PK-SIM recently, and I need to use the parameter AUC0-24h for calculation. Do AUC_tD1-tD2 represent AUC0-24h in the pharmacokinetic parameters of population simulation?

**Comment by Yuri05** [comment]
**AUC_tD1-tD2** is the AUC **from the administration time of the first dose until the administration time of the second dose**
(s. the documentation: https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations#pk-parameters)
So if your administration schema is "once daily" - it would give you AUC0-24h. Otherwise: no
